.clippy.toml

```diff
@@ -10,8 +10,29 @@ trivial-copy-size-limit = 16
 # END LINEBENDER LINT SET
 
 # Don't warn about these identifiers when using clippy::doc_markdown.
-# (`".."` means the default configuration built-in to Clippy)
-doc-valid-idents = ["FizzBuzz", "MathML", "RustNL", ".."]
+doc-valid-idents = [
+    "FizzBuzz",
+    "MathML",
+    "RustNL",
+    # TODO: Get this allowed in upstream clippy?
+    "BiDi",
+    # For better or for worse, we don't treat pass names as identifiers in our docs.
+    # TODO: Maybe we should?
```

Comment:
cc @PoignardAzur, thoughts? I'm not especially bothered either way.

I see that on https://docs.rs/masonry/latest/masonry/doc/doc_05_pass_system/index.html, there's a bit of inconsistency about whether they're code blocks (`update_widget_tree`), unstyled (layout), or bolded (**compose**).